### PR TITLE
add cmdref docs

### PR DESCRIPTION
**Pull request by ii2day**

### CODE REVIEW

File: docs/cmdref/kdoctor-agent.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    --config-dir    string    config file path (default /tmp/config-map/conf.yml)
+    --app-mode      bool      agent running mode ,when using app mode, the agent only provides an HTTP and HTTPS server for testing (default false) 
+    --tls-insecure  bool      The HTTP server skips TLS authentication (default true) 
+    --tls-ca-cert   string    The CA certificate path, which is used by the agent to generate the signing certificate  (default /etc/tls/ca.crt) 
+    --tls-ca-key    string    The CA key path, which is used by the agent to generate the signing certificate  (default /etc/tls/ca.key) 
+```
+
+### ENV
+
+```
```
Comment (weizhoublue) [inline docs/cmdref/kdoctor-agent.md:21]: {"ENV_ENABLED_METRIC", "false", &AgentConfig.EnableMetric},
	{"ENV_METRIC_HTTP_PORT", "", &AgentConfig.MetricPort},
	{"ENV_AGENT_HEALTH_HTTP_PORT", "5710", &AgentConfig.AgentHealthPort},
	{"ENV_GOPS_LISTEN_PORT", "", &AgentConfig.GopsPort},
	{"ENV_WEBHOOK_PORT", "", &AgentConfig.WebhookPort},
	{"ENV_PYROSCOPE_PUSH_SERVER_ADDRESS", "", &AgentConfig.PyroscopeServerAddress},
	{"ENV_POD_NAME", "", &AgentConfig.PodName},
	{"ENV_POD_NAMESPACE", "", &AgentConfig.PodNamespace},
	{"ENV_GOLANG_MAXPROCS", "8", &AgentConfig.GolangMaxProcs},
	{"ENV_AGENT_GRPC_LISTEN_PORT", "3000", &AgentConfig.AgentGrpcListenPort},
	{"ENV_AGENT_APP_HTTP_PORT", "80", &AgentConfig.AppHttpPort},
	{"ENV_AGENT_APP_HTTPS_PORT", "443", &AgentConfig.AppHttpsPort},
	{"ENV_ENABLE_AGGREGATE_AGENT_REPORT", "false", &AgentConfig.EnableAggregateAgentReport},
	{"ENV_AGENT_REPORT_STORAGE_PATH", "", &AgentConfig.DirPathAgentReport},
	{"ENV_CLEAN_AGED_REPORT_INTERVAL_IN_MINUTE", "10", &AgentConfig.CleanAgedReportInMinute},
	{"ENV_CLUSTER_DNS_DOMAIN", "cluster.local", &AgentConfig.ClusterDnsDomain},
	{"ENV_LOCAL_NODE_IP", "", &AgentConfig.LocalNodeIP},
	{"ENV_LOCAL_NODE_NAME", "", &AgentConfig.LocalNodeName},

**Comment by weizhoublue** [comment]
add link in doc/mkdocs.yaml

File: docs/reference/kdoctor-controller.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    --configmap-daemonset-template    string  The configmap daemonset template file path (default /tmp/configmap-app-template/daemonset.yml)
+    --configmap-pod-template          string  The configmap pod template file path (default /tmp/configmap-app-template/pod.yml)
+    --configmap-service-template      string  The configmap service template file path (default /tmp/configmap-app-template/service.yml)
+    --configmap-ingress-template      string  The configmap ingress template file path (default /tmp/configmap-app-template/ingress.yml)
+```
+
+### ENV
+
+```
+    ENV_LOG_LEVEL                                  log level (DEBUG|INFO|ERROR)
```
Comment (weizhoublue) [inline docs/reference/kdoctor-controller.md:27]: 用 table， 正式点

File: docs/reference/kdoctor-controller.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+This page describes CLI options and ENV of kdoctor-controller.
+
+## kdoctor-controller daemon
+
+Run the kdoctor controller daemon.
+
+### Options
+
+```
+    --config-dir        string    config file path (default /tmp/config-map/conf.yml)
```
Comment (weizhoublue) [inline docs/reference/kdoctor-controller.md:12]: 用 table

File: docs/reference/kdoctor-agent.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+|---------------------|--------|--------------------------|--------------------------------------------------------------------------------------------------------|
+| --config-dir        | string | /tmp/config-map/conf.yml | config file path.                                                                                      |
+| --app-mode          | bool   | false                    | agent running mode ,when using app mode, the agent only provides an HTTP and HTTPS server for testing. |
+| --tls-insecure      | bool   | true                     | The HTTPS server skips TLS authentication.                                                             |
+| --tls-ca-cert       | string | /etc/tls/ca.crt          | The CA certificate path, which is used by the agent to generate the signing certificate.               |
+| --tls-ca-key        | string | /etc/tls/ca.key          | The CA key path, which is used by the agent to generate the signing certificate.                       |
+| --task-kind         | string | ""                       | The kind of task.                                                                                      |
```
Comment (weizhoublue) [inline docs/reference/kdoctor-agent.md:18]: 如有必要， 配置项，除了缺省值，都应该 提下 取值 范围

Comment (ii2day) [reply docs/reference/kdoctor-agent.md]: task-kind 增加了 values

**Pull request merged**